code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -69,7 +69,10 @@
 
 /obj/item/reagent_containers/blood/lizard
 	blood_type = "L"
-
+// yogs start -- gorilla people
+/obj/item/reagent_containers/blood/gorilla
+	blood_type = "G"
+// yogs end
```

Comment:
Can you dump this class definition into ``/yogstation/code/modules/reagents/reagent_containers/blood_pack.dm``?